docs/deployment.md

```diff
@@ -51,13 +53,13 @@ TODO Compile complete starter template somewhere in this repo...
 
 ## Development environment
 
-1) Get the main server running using the Readme here: https://github.com/pol-is/polisServer
+1) Get the main server running using the Readme here: [`/server`][dir-server]
    * This includes instructions on setting up a local postgres database
-2) Get the math server running using the Readme here: https://github.com/pol-is/polisMath
-3) Build client repo assets using the instructions in the repsective Readmes:
-   * https://github.com/pol-is/polisClientParticipation
-   * https://github.com/pol-is/polisAdmin
-   * https://github.com/pol-is/polisReport
+2) Get the math server running using the Readme here: [`/math`][dir-math]
+3) Build client repo assets using the instructions in the respective Readmes:
+   * [`/clientParticipation`][dir-participation]
```

Comment:
I'm not familiar with the [dir-participation] URL format. Can it be extended to point all the way to the READMES? It would save several clicks.

Reply:
It's just a reference, and let's link be defined elsewhere in doc (earlier in doc, in this case). Helps links not break formatting flow of plaintext so much. Prob looks a little more confusing here because the "link" is to a directory instead of an URL like it normally is